Week-9/to-do-app/src/container/ToDoList/index.jsx

```diff
@@ -0,0 +1,45 @@
+import FormTodo from "../../components/FormToDo";
+import TodoListItem from "../../components/ToDoListItem";
+import { useState } from "react";
+
+const ToDoList = () => {
+  const [toDos, setToDos] = useState([]);
+
+  const handleNewToDoCreation = (text) => {
+    setToDos((toDos) => [...toDos, { text, id: toDos.length }]);
+  };
+
+  const handleToDoToggleCompletion = (id) => {
+    const newToDos = [...toDos];
+    newToDos.find((todo) => todo.id === id).isDone = !newToDos.find(
```

Comment:
@apandyamob Why this much complexity ? Can you add the data model you are using ?
Will todo have { id; text; isDone }

Can you simplify this ? Is it just isDone flag update to true ?

Reply:
@apandyamob one more thing, why two images for the tick mark. Is there any difference ?

Reply:
> @apandyamob one more thing, why two images for the tick mark. Is there any difference ?

both images are having different resolutions, images are used in the manifest file.

Reply:
> @apandyamob Why this much complexity ? Can you add the data model you are using ? Will todo have { id; text; isDone }
> 
> Can you simplify this ? Is it just isDone flag update to true ?

Upon clicking on todo text, I am toggling isDone value to true/false.. I have optimized the logic. Thanks @RajeshSivanesan